installation/proc-add-custom-app-config-file-ocp.adoc

```diff
@@ -0,0 +1,86 @@
+[id="proc-add-custom-app-config-file-ocp_{context}"]
+= Adding a custom application configuration file to {ocp-short}
+
+To change the configuration of your {product} instance, you must do the following:
+
+* Add a custom application configuration file to {ocp-short} and reference it in the Custom Resource (CR). In {ocp-short}, you can use the following example as a base template to create a ConfigMap such as `app-config-rhdh.yaml`:
++
+[source,yaml,subs="attributes+"]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: app-config-rhdh
+data:
+  "app-config-rhdh.yaml": |
+    app:
+      title: {product}
+      baseUrl: https://backstage-developer-hub-my-ns.apps.ci-ln-vtkzr22-72292.origin-ci-int-gce.dev.rhcloud.com
+    backend:
+      auth:
+        keys:
+          - secret: "${BACKEND_SECRET}"
+      baseUrl: https://backstage-backstage-sample-my-ns.apps.ci-ln-vtkzr22-72292.origin-ci-int-gce.dev.rhcloud.com
+      cors:
+        origin: https://backstage-backstage-sample-my-ns.apps.ci-ln-vtkzr22-72292.origin-ci-int-gce.dev.rhcloud.com
+----
++
+* Use the mandatory backend auth key for {product} to reference an environment variable defined in an {ocp-short} Secret.
+* Set the external URL of your {product} instance in the `app.baseUrl`, `backend.baseUrl` and `backend.cors.origin` fields of the application configuration. By default, the URL is similar to the following example:
+`pass:c[https://backstage-<CUSTOM_RESOURCE_NAME>-<NAMESPACE_NAME>.<OPENSHIFT_INGRESS_DOMAIN>;]`.
+** You can use the `oc get ingresses.config/cluster -o jsonpath='{.spec.domain}'` command to display your ingress domain. If you want to use a different host or sub-domain, customize the `Custom Resource spec.application.route field` and adjust the application configuration accordingly.
+
+[NOTE]
+You are responsible for protecting your {product} installation from external and unauthorized access. Manage the backend auth key like any other secret. Meet strong password requirements, do not expose it in any configuration files, and only inject it into configuration files as an environment variable.
```

Comment:
```suggestion
[NOTE]
====
You are responsible for protecting your {product} installation from external and unauthorized access. Manage the backend auth key like any other secret. Meet strong password requirements, do not expose it in any configuration files, and only inject it into configuration files as an environment variable.
====
```

We generally use this [admonition syntax](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions) per the RH supplementary style guide.